pkg/transformer/kubernetes/k8sutils.go

```diff
@@ -901,7 +901,24 @@ func FormatEnvName(name string) string {
 	if len(envName) > 63 {
 		envName = envName[len(envName)-63:]
 	}
```

Comment:
I think that we should delete this bloc here

Reply:
applied

Reply:
from k8sutils_test.go this test gets last 63 characters
```
{
			name: "check that ./ is removed",
			args: args{
				name: "abcdefghijklnmopqrstuvxyzabcdefghijklmnopqrstuvwxyzabcdejghijkl$Hereisadditional",
			},
			want: "rstuvxyzabcdefghijklmnopqrstuvwxyzabcdejghijkl$Hereisadditional",
		},
	```	
and gets affected by this removed code
```
if len(envName) > 63 {
		envName = envName[len(envName)-63:]
	}
```